docs/walkthrough/finetuning-models.md

```diff
@@ -1,10 +1,128 @@
 ---
-sidebar_position: 2
+sidebar_position: 4
 ---
 
 # Finetuning Models
 
 <!-- THE MARKDOWN (.md) FILE IS GENERATED FROM THE NOTEBOOK (.ipynb) FILE -->
+The large pretrained base models are already very helpful, but
+with finetuning smal and large language models can be made even more helpful.
 
-Finetuning a model...
+Finetunig is especially helpful for use cases that require private internal
+data or for use cases that require domain specific knowledge that wasn't
+available on the internet.
 
+In Substratus the Model resource allows you to specify a `baseModel` and
+a `trainingDataset` to make it very easy to create new finetuned models.
+
+To summarize: 
+`finetunedModel = training_image(baseModel, trainingDataset)`. 
```

Comment:
Better described with a diagram (we have one in the architecture section that shows this).

Reply:
why not both?